### ISSUE TITLE
Add secondary DICOM server parameter for IDC viewer handler

### ISSUE THREAD

**Issue by fedorov** [ImagingDataCommons/SlicerIDCBrowser#24]
This is follow up on https://github.com/vkt1414/slicer-idc-viewer/issues/2.

@pieper I thought about this after the discussion yesterday. I understand the use case better now, and I think adding the feature that would allow parameterizing `slicer://idc-viewer/` with a DICOMweb endpoint is a specific functionality we could explore towards implementing the workflow in the issue above. I initially thought it would be better to have this handler in mpReview, but since most of the logic will be in IDCBrowser, it makes more sense to implement it here.

I think the functionality would be the following:
* `idc-viewer` handler is passed a list of `SeriesInstanceUID`s as one parameter
* and the GCP DICOMweb URL as the second parameter
* handler uses `idc-index` to resolve `StudyInstanceUID` for each of the `SeriesInstanceUID`s and fetches all of the available series in those studies using DICOMweb from the secondary server

We should reuse the functionality from SlicerDICOMWeb, similar to how it is done in mpReview in https://github.com/SlicerProstate/mpReview/blob/master/mpReview.py#L534-L543 and https://github.com/SlicerProstate/mpReview/blob/master/mpReview.py#L1814.

Is this consistent with what you were looking for @pieper? I was confused by the other issue since it was very broad.

I think we should look into this only after the basic handler is functional.

**Comment by pieper** [comment]
Yes, thanks @fedorov that sounds like a very useful feature to support the use case we discussed.

**Comment by pieper** [comment]
Do we have a place where the slicer:// url parameter schema is documented where we can add info about these new fields?

**Comment by fedorov** [comment]
No, we don't. I think we can start discussing this whenever @vkt1414 has the PR opened. But for now I want to focus on TotalSegmentator project and make a push to try to get it done before the PW. I think we can use PW project for ironing out details like the schema and this new functionality.
